scripts/cover/coverGcov.sh

```diff
@@ -1,50 +1,46 @@
 #!/usr/bin/env bash
 set -e
-
 # set HOOT_HOME if not set
 if [ -z "$HOOT_HOME" ]; then
     HOOT_HOME=~/hoot
 fi
-
 # generate gcov files from gcda files
 cd $HOOT_HOME
-
+# create the gcov output folder
+if [ ! -d gcov ]; then
+    mkdir gcov
+else
+    rm -f gcov/*
+fi
```

Comment:
All `gcov` output files are assembled in one folder that isn't `$HOOT_HOME`.

Reply:
Nice.